e2e/call-done-twice/__tests__/index.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+describe('`done()` called more than once', () => {
```

Comment:
can you add tests for calling `done` multiple times in hooks as well?

Reply:
done!